src/retail-data-visualization/Makefile

```
@@ -0,0 +1,11 @@
+build:
```

```suggestion
build: install-infinity
```
this will automatically line 4, user doesn't have to do any extra step then